server/src/controllers/parcels/changeStatus.ts

```diff
@@ -0,0 +1,13 @@
+import { changeStatusQuery } from '../../database/queries';
+
+const changeStatus = async (req:any, res:any) => {
+  const { userId: sellerId } = req;
+  const { id } = req.params;
+  const ParcelId = +id;
+  const { rows, rowCount } = await changeStatusQuery({ sellerId, ParcelId });
```

Comment:
can you explain more how the rowCount will be the falsy value here? 
here we will set update to our table not get an exist data from table.